terraform/modules/static-frontend/cdn.tf

```diff
@@ -0,0 +1,130 @@
+# Access identity, policy and document for the CDN
+
+locals {
+  media_s3_origin_id = "S3-${var.project}-${var.environment}-assets-${local.unique_string}"
+}
+
+resource "aws_cloudfront_origin_access_identity" "origin_access_identity" {
+  comment = "CloudFront Origin Access Identity for ${var.project}-${var.environment}-assets-${local.unique_string}"
+}
+
+data "aws_iam_policy_document" "cdn" {
+  statement {
+    actions = ["s3:GetObject"]
+    resources = ["${aws_s3_bucket.bucket.arn}/*"]
+
+    principals {
+      type        = "AWS"
+      identifiers = [aws_cloudfront_origin_access_identity.origin_access_identity.iam_arn]
+    }
+  }
+}
+
+resource "aws_s3_bucket_policy" "cdn" {
```

Comment:
this is the bucket policy. it defines specific rules for what permissions are allowed and/or denied on a resource. this one allows the cdn to get objects from the bucket